code/modules/hivemind/wires.dm

```diff
@@ -381,20 +381,20 @@
 		if(ishuman(subject))
 			var/mob/living/L = subject
 
-			if(GLOB.hive_data_bool["prevent_gibbing_dead"])
-			//We we dont touch the dead via are controler we dont want to pk people form the round
+			if(!GLOB.hive_data_float["gibbing_warning_timer"]) //If the value is set to 0 (the default) we don't touch player humans
 				return
 
-			//if our target has cruciform, let's just leave it
-			if(is_neotheology_disciple(L))
+			var/timer = GLOB.hive_data_float["gibbing_warning_timer"] SECONDS //If we've continued, then there's a value there and we want it in seconds
+
+			if(is_neotheology_disciple(L)) //If our target has a cruciform, we don't touch it
 				return
 
-			for(var/obj/item/W in L)
-				L.drop_from_inventory(W)
-			var/M = pick(/mob/living/simple_animal/hostile/hivemind/himan, /mob/living/simple_animal/hostile/hivemind/phaser)
-			new M(loc)
+			visible_message("Wires begin to wreathe around [L], starting the process of converting them into part of the hivemind.") //We tell people to get them off the wires
+			addtimer(CALLBACK(src, .proc/assimilate_human, L), timer)
```

Comment:
```suggestion
			timer_id = addtimer(CALLBACK(src, .proc/assimilate_human, L), timer, TIMER_STOPPABLE)
```

Reply:
Our weeds are going to be cut and destroyed. Github is being annoying but you also need to define a basic var/timer_id at the top and deltimer(timer_id) for both Destroy() and die_off(). Should only be:

```
/obj/effect/plant/hivemind/die_off()
	if(timer_id)
		deltimer(timer_id)
	return ..()
```